docs/multi-homing.md

```diff
@@ -0,0 +1,118 @@
+
+## Configuring secondary networks
+To allow pods to have multiple network interfaces, the user must provide the
+configurations specifying how to connect to these networks; these
+configurations are defined in a CRD named `NetworkAttachmentDefinition`.
+
+Below you will find example attachment configurations for each of the current
+topologies OVN-K allows for secondary networks.
+
+**NOTE**: currently, all the secondary networks **only** allow for east/west
+traffic.
+
+### Routed - layer 3 - topology
+This topology is a simplification of the topology for the cluster default
+network - but without egress.
+
+There is a logical switch per node - each with a different subnet - and a
+router interconnecting all the logical switches.
+
+The following net-attach-def configures the attachment to a routed secondary
+network.
+
+```yaml
+apiVersion: k8s.cni.cncf.io/v1
+kind: NetworkAttachmentDefinition
+metadata:
+  name: l3-network
+  namespace: ns1
+spec:
+  config: |2
+    {
+            "cniVersion": "0.3.1",
+            "name": "l3-network",
+            "type": "ovn-k8s-cni-overlay",
+            "topology":"layer3",
+            "subnets": "10.128.0.0/16/24",
+            "mtu": 1300,
+            "netAttachDefName": "ns1/l3-network"
+    }
+```
+
+**NOTE**
+- the `netAttachDefName` parameter **must** match <namespace>/<net-attach-def name>
+  of the surrounding object.
+- the `subnets` attribute is a comma separated list of subnets.
+- the `subnets` attribute indicates both the subnet across the cluster, and per node.
+  The example above means you have a /16 subnet for the network, but each **node** has
+  a /24 subnet.
+
+### Switched - layer 2 - topology
+This topology interconnects the workloads via a cluster-wide logical switch.
+
+The following net-attach-def configures the attachment to a layer 2 secondary
+network.
+
+```yaml
+apiVersion: k8s.cni.cncf.io/v1
+kind: NetworkAttachmentDefinition
+metadata:
+  name: l2-network
+  namespace: ns1
+spec:
+  config: |2
+    {
+            "cniVersion": "0.3.1",
+            "name": "l2-network",
+            "type": "ovn-k8s-cni-overlay",
+            "topology":"layer2",
+            "subnets": "10.100.200.0/24",
+            "mtu": 1300,
+            "netAttachDefName": "ns1/l2-network"
+    }
+```
+
+## Pod configuration
+The user must specify the secondary network attachments via the
+`k8s.v1.cni.cncf.io/networks` annotation.
+
+The following example provisions a pod with two secondary attachments, one for
+each of the attachment configurations presented in
+[Configuring secondary networks](#configuring-secondary-networks).
+
+```yaml
+apiVersion: v1
+kind: Pod
+metadata:
+  annotations:
+    k8s.v1.cni.cncf.io/networks: l3-network,l2-network
+  name: tinypod
+  namespace: ns1
+spec:
+  containers:
+  - args:
+    - pause
+    image: k8s.gcr.io/e2e-test-images/agnhost:2.36
+    imagePullPolicy: IfNotPresent
+    name: agnhost-container
+```
+## Limitations
```

Comment:
Can I request a specific IP using the `networks` annotation or is it not supported yet?

Reply:
the OVN-K secondary networks do **not** invoke the IPAM plugin, as such, no one will act upon the requested attachment IP afaict.

So no.